support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -298,7 +301,14 @@ EventsAndTriggers:
         Value: 2.5
     Events:
       - ObservationAhC
-  # Never terminate... run until something fails!
+  # Added stop after time = 1000 to get ~2 orbits; number can be increased
+  # or decreased based on desired number of orbits
+  - Trigger:
+      TimeCompares:
+        Comparison: GreaterThan
+        Value: 1000
+    Events:
+      - Completion
```

Comment:
For now, I don't think we want to stop our evolutions early. Once we have more logic for automatically stopping, doing an ID solve, and then restarting, then we can add something like this back in.